src/nextjournal/clerk/viewer.cljc

```diff
@@ -516,7 +517,9 @@
 
 (defn with-block-viewer [doc {:as cell :keys [type]}]
   (case type
-    :markdown [(with-viewer `markdown-viewer (process-sidenotes doc (:doc cell)))]
+    :markdown [(with-viewer `markdown-viewer
+                 {:nextjournal.clerk/opts (select-keys doc [:index :file :bundle? :out-path])}
```

Comment:
👆 Not super happy with the selec-keys here, but in this case it's harder to pass down doc options along the markdown tree. 

Options will land in the presentation:

```clj
(-> @nextjournal.clerk.webserver/!doc
      viewer/notebook
      viewer/present
      viewer/->value
      :blocks)
=>
[{:nextjournal/value [:div.markdown-viewer
                      [:p
                       [:<> "what a piece "]
                       [#viewer-evalnextjournal.clerk.render/inspect-presented
                        {:nextjournal/value [:img.inline
                                             {:src "_blob//Users/amantini/dev/clerk/images/vera.jpg", :alt ""}],
                         :nextjournal/viewer {:name nextjournal.clerk.viewer/html-viewer,
                                              :render-fn #viewer-fnidentity,
                                              :hash "5du5eveS8XMqMDMYH5rpQ1bqryEJdg"}}]
                       [:<> " ahoi"]]],
  :nextjournal/opts {:file "/Users/amantini/dev/clerk/notebooks/scratch/debug.clj"},
  :nextjournal/viewer {:name nextjournal.clerk.viewer/html-viewer-,
                       :render-fn #viewer-fnidentity,
                       :hash "5du1K91pV2nRmpCG3vM8fBcDsgHWKs"}}]
```
unless we explicitely dissoc them in `process-blocks`, but I'd find that even worse.